config.json

```diff
@@ -136,7 +136,13 @@
       "core": false,
       "difficulty": 4,
       "slug": "luhn",
-      "topics": null,
+      "topics": [
+        "booleans",
+        "loops",
+        "mathematics",
+        "strings",
+        "type_conversion"
```

Comment:
Great suggestions! The only one I'm not sure about is `type_conversion`. How would you say that relates to this exercise? 

I would also maybe add the `algorithms` tag since this exercise is to do with implementing a specific algorithm :)

Reply:
The `algorithms` tag is a great idea! I will add that one. 

Regarding the `type_conversion`, I added it because it's necessary to convert the String representation of the number (since that is the parameter the function takes in the skeleton) to an integer array (or other integer-based representation) in order to do the processing and validate.

Let me know if this reasoning sounds solid to you. If not, I can simply remove it (:

Reply:
Sure, that sounds fair :) Thanks for explaining! :) Happy to merge it once you've added the `algorithms` tag then! :)

Reply:
Ok! It's done now :)